### PR TITLE
feat: add support for Exodus wallet

### PR DESCRIPTION
**Pull request by olistic**
## Summary

This PR adds support for the Exodus wallet.

### CODE REVIEW

File: packages/use-solana/src/providers.tsx [hunk 1]
```diff
@@ -101,6 +104,14 @@ export const DEFAULT_WALLET_PROVIDERS: WalletProviderMap<
 
     isInstalled: () => window.solong !== undefined,
   },
+  [DefaultWalletType.Exodus]: {
+    name: "Exodus",
+    url: "https://www.exodus.com/browser-extension",
+    icon: EXODUS,
+    makeAdapter: () => new SolanaWalletAdapter(new ExodusWalletAdapter()),
+    isInstalled: () => window.exodus?.solana !== undefined,
+    isMobile: true,
```
Comment (olistic) [inline packages/use-solana/src/providers.tsx:113]: I noticed that many similar extensions have this flag (`isMobile`) set. What does it mean? Trying to figure out whether we need it for Exodus or not.

**Comment by olistic** [comment]
@macalinao @michaelhly This is ready for review. Looking forward to your feedback!

**Comment by michaelhly** [comment]
@olistic would you mind fixing the failures from the CI runs?

**Comment by olistic** [comment]
> would you mind fixing the failures from the CI runs?

@michaelhly CI doesn't seem to be running on this PR, as it says it needs a maintainer to approve the workflow. However, when I run `yarn test` locally I get what it looks like an unrelated error:

```
yarn test
 PASS  packages/solana-contrib/src/transaction/parseTransactionLogs.test.ts
 FAIL  packages/stableswap-sdk/src/calculator/amounts.unit.test.ts
  ● Test suite failed to run

    Cannot find module '@saberhq/token-utils' from '/Users/oli/Projects/Forks/saber-common/packages/stableswap-sdk/src/calculator'

      at resolveSync (../../../.yarn/berry/cache/resolve-patch-bad885c6ea-8.zip/node_modules/resolve/lib/sync.js:111:15)

 FAIL  packages/stableswap-sdk/src/calculator/curve.unit.test.ts
  ● Test suite failed to run

    Cannot find module '@saberhq/token-utils' from '/Users/oli/Projects/Forks/saber-common/packages/stableswap-sdk/src/calculator'

    Require stack:
      packages/stableswap-sdk/src/calculator/curve.ts
      packages/stableswap-sdk/src/calculator/curve.unit.test.ts

      at resolveSync (../../../.yarn/berry/cache/resolve-patch-bad885c6ea-8.zip/node_modules/resolve/lib/sync.js:111:15)

Test Suites: 2 failed, 1 passed, 3 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        1.748 s, estimated 2 s
```

File: packages/use-solana/src/providers.tsx (hunk 1 above)
Comment (michaelhly) [reply packages/use-solana/src/providers.tsx:113]: does your adapter work on a mobile browser?

**Comment by michaelhly** [comment]
@olistic they're still failing. Can you make sure you run these steps locally?
https://github.com/saber-hq/saber-common/blob/2c5ba9ce7454854d2019333ed6b36f3b1e8f37e2/.github/workflows/main.yml#L9

**Comment by olistic** [comment]
@michaelhly Ran and fixed!

**Review by michaelhly — APPROVED** [review]
LGTM!

**Pull request merged**